docs/documentation/indexing/create_index.mdx

```diff
@@ -18,6 +18,8 @@ USING bm25 (id, description, category, rating, in_stock, created_at, metadata, w
 WITH (key_field='id');
 ```
 
+For details on configuring and tokenizing individual fields and data types, see the [Field Configuration](documentation/indexing/field_options) section.
+
```

Comment:
The syntax section below also doesn't suggest that you can add any other options to the `WITH` block (only the `key_field`). It might be good to add this same link there as well?